packages/table/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@blueprintjs/table",
-  "version": "1.29.0",
+  "version": "1.29.1",
```

Comment:
these two package.json changes also don't belong here... perhaps you branched off something other than the latest `master`?